### PR TITLE
adding 'Troubleshooting' section to README

### PR DESCRIPTION
**Pull request by aaronbauman**
Copying some troubleshooting tips from this thread. https://github.com/pantheon-systems/quicksilver-examples/issues/24

### CODE REVIEW

**Comment by greg-1-anderson** [comment]
Just a clarification on `create` vs. `import`. Did you have trouble with importing via the dashboard (at site create time), or were you importing via `drush @pantheon.mysite.dev archive-restore`?

**Comment by aaronbauman** [comment]
I imported via the dashboard from a remotely-hosted drush archive

**Comment by greg-1-anderson** [comment]
Ok.  +1 for this PR, then.  Thanks.

File: README.md [hunk 1]
```diff
@@ -67,3 +67,15 @@ $ terminus help workflows
 
 The `list` and `show` commands will allow you to explore previous workflows and their Quicksilver operations. The `watch` command is a developers best friend: it will set up Terminus to automatically "follow" the workflow activity of your site, dumping back any Quicksilver output along with them.
 
+## Troubleshooting ##
+- `pantheon.yml` goes in your drupal root, ie. your ~/code directory
```
Comment (joshkoenig) [inline README.md]: The term "drupal root" is platofrm specific. It should be "site root".

File: README.md [hunk 2]
```diff
@@ -67,3 +67,15 @@ $ terminus help workflows
...
+- `private` also goes in your drupal root, and is git-ignored by default. You must `--force` add any and all changes
```
Comment (joshkoenig) [inline README.md]: > is git-ignored by default

`~code/private` should not be gitignored by default. That's the explicit place on the platform where we document customers commit things they do not want exposed on the web. It's a bug with us upstream that this is listed in our Drupal 8 stock `.gitignore` file and we should deal with it as such.

No `--force` pushing should be necessary to get Quicksilver working.

**Comment by aaronbauman** [comment]
OK, updated PR.

Is there a public issue I can track about the ignore problem?
I couldn't find one.

**Comment by greg-1-anderson** [comment]
I created a PR for the .gitignore problem: https://github.com/pantheon-systems/drops-8/pull/122

This will go in with Drupal 8.0.3. The next Drupal 8 release window is [Wednesday February 3](https://groups.drupal.org/node/508572).

**Comment by greg-1-anderson** [comment]
Drupal 8.0.3 has been deployed to the Pantheon environment.  Update, and you shouldn't have any trouble with your .gitignore excluding the 'private' directory any longer.

**Comment by joshkoenig** [comment]
Looks like we have a handle on the issue with imported sites, and that should be fixed shortly. 

I still think a TROUBLESHOOTING section is a good idea.

**Pull request merged**